configuration/sitemaps.md

```diff
@@ -130,23 +130,27 @@ The following element types can be used in a Sitemap definition file.
 Data presented by Sitemap elements will almost always originate from a referenced item.
 Each Item is of a certain Item type, for example `Switch`, `Number` or `String`.
 
-While not all combinations are meaningful, items of one datatype can be linked to different Sitemap element types.
+While not all combinations are meaningful, items of one datatype may be linked to different Sitemap element types.
 This provides the flexibility to present items in the way desired in your home automation user interface.
 
+<!-- TODO: an example of the above would be useful here -->
+
```

Comment:
Okay - I will look at this.

Reply:
What I meant is that a little explanation would be helpful.  How about adding a sentence just above the TODO as follows:
"For example, a Temperature Item may be rendered as text to display the temperature in degrees C.  It may also be rendered as a Slider Element, allowing a user to set the temperature in a room to a specific setting."

Reply:
🙂👍